docs/config.json

```
@@ -1385,6 +1385,10 @@
             {
               "title": "PostgreSQL",
               "slug": "/database-access/auto-user-provisioning/postgres/"
+            },
+            {
+              "title": "Database object permissions",
```

```suggestion
              "title": "Database Object Permissions",
```

For consistency with other docs titles (we use title case)